### PR TITLE
[Api-draft] add README.zh.md for new api

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by Hisoka-X** [comment]
I will submit english version later.

File: seatunnel-connectors/seatunnel-connectors-seatunnel/README.zh.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## 目的
+Because SeaTunnel design new API for connectors, 所以通过这篇文章来介绍新的接口以及新的代码结构，方便开发者快速的帮助新API和翻译层完善，以及开发出新的Connecotor.
+## 代码结构
+现阶段所有相关代码保存在`api-draft`分支上。
+为了和老的代码分开，方便现阶段的并行开发，以及降低merge的难度。我们为新的执行流程定义了新的模块
+### Example
+我们已经在`seatunnel-examples`中准备好了新版本的可本地执行Example程序，直接调用`seatunnel-flink-examples`或`seatunnel-spark-examples`中的`SeaTunnelApiExample`即可。这也是本地开发Connector经常会用到的调试方式。
```
Comment (ruanwenjun) [inline seatunnel-connectors/seatunnel-connectors-seatunnel/README.zh.md]: 我等一下会新添加两个测试module`seatunnel-flink-new-connector-example`,`seatunnel-spark-new-connector-example`用于新的connector的本地测试

Comment (ruanwenjun) [reply seatunnel-connectors/seatunnel-connectors-seatunnel/README.zh.md]: Done.

Comment (ruanwenjun) [inline seatunnel-connectors/seatunnel-connectors-seatunnel/README.zh.md]:
```suggestion
我们已经在`seatunnel-examples`中准备好了新版本的可本地执行Example程序，直接调用`seatunnel-flink-new-connector-example`或`seatunnel-spark-new-connector-example`中的`SeaTunnelApiExample`即可。这也是本地开发Connector经常会用到的调试方式。
```

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

**Pull request merged**